android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -530,7 +530,12 @@ fun ReadableMap.toSubtitleTrack(): SubtitleTrack? {
 /**
  * Converts any subtitle format name in its mime type representation.
  */
-private fun String.toSubtitleMimeType(): String = "text/$this"
+private fun String.toSubtitleMimeType(): String = when (this) {
+    "vtt" -> "text/vtt"
```

Comment:
not needed, same as the else branch, right?

Reply:
True, fixed in cc2198e